scripts/publish.sh

```diff
@@ -25,9 +30,6 @@ fi
 # check param, if it's set (monorepo) we check if it's published before proceeding
 if [[ -n "$1" ]]; then
   # check if module is published
-  PACKAGE_NAME=$(jq --raw-output .name package.json)
-  LATEST_PACKAGE_VERSION=$(npm view "$PACKAGE_NAME" dist-tags --workspaces false --json | jq --raw-output --arg tag "$PUBLISH_NPM_TAG" '.[$tag]' || echo "")
```

Comment:
I don't think we need to move this part. We can leave this here, because it's only applicable to the monorepo case anyway. At the moment there is no need to check for the latest published version of a package for the single-package repository case.

I think it's sufficient to move the `if [[ -z $YARN_NPM_AUTH_TOKEN ]]; then` block to the end, just before the publish step. Does that work?